### ISSUE TITLE
Failing to block viatepigan.com

### ISSUE THREAD

**Issue by Kauvincx** [EFForg/privacybadger#2535]
Hi,

Even though Privacy Badger is doing a pretty job at blocking pop-up redirects, it is somehow failing to block viatepigan.com, when I visit some specific domains (those which try to load viatepigan.com), and I believe this could be a bug with this extension. I had this domain set to be blocked in the extension's settings, and I already checked if my computer was infected with adware.

Privacy Badger (chrome extension) version 2019.11.18

Thanks!

**Comment by ghostwords** [comment]
Hello! What is `viatepigan.com`? Could you give us a way to reproduce this issue?

**Comment by Kauvincx** [comment]
> Hello! What is `viatepigan.com`? Could you give us a way to reproduce this issue?

Viatepigan.com is a known pop-up redirect site. Unfortunately, the only way to reproduce this issue is by visiting some of those domains I've explained in my first post. For example, I've visited uiz.io (a URL shortener) and it loads this site as a iframe (see screenshots attached), even though I've set the extension to block the site.

This is the link I've been doing some testing on (WARNING: it contains a lot of ads/popups/popunders): http://uiz.io/2iETa

![viatepigan](https://user-images.githubusercontent.com/59508271/71946253-a4a41800-319f-11ea-8bc0-4c92291df62c.JPG)
![viatepigan2](https://user-images.githubusercontent.com/59508271/71946254-a4a41800-319f-11ea-92fa-1007a714ebcb.JPG)
